cloudfront/invalidation/README.md

```diff
@@ -2,6 +2,9 @@
 
 Lambda function for invalidating CloudFront caches, triggered by SNS topic. 
 
-Message sent to SNS topic is an array of paths to invalidate, e.g. `["/path/to/invalidate", "/wildcard/path*"]`
+SNS message body is an array of paths to invalidate, e.g. 
+```
+{ "paths": ["/path/to/invalidate", "/wildcard/path*", "/catalogue/v2/works/z869w74a"] }
```

Comment:
It looks like the previous documentation was incorrect (omitting path from the incoming message), good spot fixing it.